README.md

```diff
@@ -0,0 +1,70 @@
+# D.IMS.IO
+
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/quintilesims/d.ims.io/blob/master/LICENSE)
+[![Go Report Card](https://goreportcard.com/badge/github.com/quintilesims/d.ims.io)](https://goreportcard.com/report/github.com/quintilesims/d.ims.io)
+[![Go Doc](https://godoc.org/github.com/quintilesims/d.ims.io?status.svg)](https://godoc.org/github.com/quintilesims/d.ims.io)
+
+
+## Overview
+This repository serves as a web front-end for [EC2 Container Registries](https://aws.amazon.com/ecr/).
+It provides custom authentication and an API to create and manage Docker repositories. 
+
+## Repository Management
+You must create a new repository for each image you want to host.
+Creating new repositories can be done through the [Swagger UI](https://d.ims.io/api/?url=/swagger.json).
+
+Each image must have both an `owner` and a `name`.
```

Comment:
[nit] I have a concern that mentioning `owner` here and then using `team` on the next line might trip somebody up.